src/components/AnonymousChat/PostNotificationChannelItem.tsx

```diff
@@ -99,7 +97,7 @@ const PostNotificationChannelItem: (props: MessageChannelItemProps) => React.Rea
       comments={item?.rawJson?.comments?.length}
       downvote={item?.rawJson?.downvote}
       isCommentExists={Boolean(firstComment)}
-      isMe={item?.user?.isMe || isOwnSignedPost || isOwnPost}
+      isMe={getIsMe()}
       isOwnSignedPost={isOwnSignedPost}
       message={item?.description}
       name={anonUsername}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code replaces the previous logic for determining `isMe` with a call to the new `getIsMe` function. This simplifies the code and improves readability. However, please ensure that the `getIsMe` function always returns the correct result, as it is now used in a critical part of the component.
```diff
-      isMe={item?.user?.isMe || isOwnSignedPost || isOwnPost}
+      isMe={getIsMe()}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->